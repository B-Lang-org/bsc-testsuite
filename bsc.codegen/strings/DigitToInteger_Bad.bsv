(* synthesize *)
module sysDigitToInteger_Bad();
   rule r;
      $display(digitToInteger("a"));
   endrule
endmodule
