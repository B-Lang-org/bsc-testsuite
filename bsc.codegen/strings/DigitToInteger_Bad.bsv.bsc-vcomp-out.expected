checking package dependencies
compiling DigitToInteger_Bad.bsv
code generation for sysDigitToInteger_Bad starts
Error: "DigitToInteger_Bad.bsv", line 4, column 31: (S0015)
  Bluespec evaluation-time error: digitToInteger: not a digit: `a'
  During elaboration of the body of rule `r' at "DigitToInteger_Bad.bsv", line
  3, column 9.
  During elaboration of `sysDigitToInteger_Bad' at "DigitToInteger_Bad.bsv",
  line 2, column 8.
